plugins/Ecommerce/tests/System/expected/test_ecommerceOrderWithItems_schedrep_html_tables_and_graph__ScheduledReports.generateReport_week.original.html

```diff
@@ -3408,7 +3408,7 @@ <h2 id="CustomVariables_getCustomVariables" style=" color:#0d0d0d;font-family:-a
                                                     
                                                                     <tr style=";">
                                                                 <td style="padding:17px 15px;;border-bottom:1px solid rgb(231,231,231);font-size: 15px;font-variant-numeric: tabular-nums;padding:17px 15px;color:#0d0d0d;font-family:-apple-system, BlinkMacSystemFont, &#039;Segoe UI&#039;, Roboto, Oxygen-Sans, Cantarell, &#039;Helvetica Neue&#039;, sans-serif;">
-                                                                                                                                                                        ValueIsZero - 0                                                                                                                        </td>
+                                                                                                                                                                        ValueIsZero - Others                                                                                                                        </td>
```

Comment:
@diosmosis this is bit unexpected maybe? I can see here we set the value to `0` https://github.com/matomo-org/matomo/blob/3.11.0-b1/tests/PHPUnit/Fixtures/TwoSitesEcommerceOrderWithItems.php#L68 so would have expected to see that value? Or is it because of https://github.com/matomo-org/matomo/pull/14622/files#diff-64a9dbfd70403b4ac01ee0d4cfd1894cR185 being set to 3/2 rows truncation and it's actually grouping values here? Actually all good... I can see here (https://github.com/matomo-org/matomo/pull/14622/files#diff-05a120dee88125d268f6cd302e02d6bdL161) there are already 2 other rows in the subtable so it is expected that it is grouped.